LibTest/ffi/Bool/Bool_A01_t02.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Represents a native bool in C.
+///
+/// Bool is not constructible in the Dart code and serves purely as marker in
+/// type signatures.
+///
+/// @description Checks that this type represents a native bool in C.
+/// @author sgrekhov22@gmail.com
+
+import "dart:ffi";
+import "package:ffi/ffi.dart";
+import "../../../Utils/expect.dart";
+
+void main() {
+  Pointer<Bool> p1 = calloc<Bool>();
+  Pointer<Int16> p2 = new Pointer<Int16>.fromAddress(p1.address);
```

Comment:
All the tests below rely on a little endian encoding of the number: the least significant byte is used if only one byte is read.

This happens to succeed right now because we do not support any big endian architectures in Dart at the moment.

Instead, use Uint8 to avoid big/little endian issues in this test.